docs/development/design-files.md

```
@@ -0,0 +1,53 @@
+# Design Files
+
+UI design files (`.pen`, [Pencil](https://pencil.app) format) are stored under `designs/screens/`.
+
+## File Layout
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)